circle.yml

```diff
@@ -56,25 +56,25 @@ dependencies:
         -e PKG_VERSION=${PKG_VERSION} \
         -e PKG_RELEASE=${PKG_RELEASE} \
         ${DISTRO} build
-  post:
-    - docker-compose -f ~/st2box/docker-compose.yaml pull
+#  post:
+#    - docker-compose -f ~/st2box/docker-compose.yaml pull
 
 test:
```

Comment:
Yes, that's the container used for testing. We don't need to pull. Verified alreadt https://github.com/StackStorm/st2chatops/pull/60